components/Account/enhancers.js

```diff
@@ -0,0 +1,78 @@
+import { graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+import { BOOKMARKS_COLLECTION_NAME } from '../Bookmarks/fragments'
+
+export const userDetailsFragment = `
+  fragment Details on User {
```

Comment:
```suggestion
  fragment PhoneAndAddressOnUser on User {
    id
```
The more explicit name makes it easier to find usages in the code base. Adding the id to the fragment ensure it's merged into the right cache always.